swatch-core/src/main/java/org/candlepin/subscriptions/db/model/Host.java

```diff
@@ -95,7 +95,7 @@ public class Host implements Serializable {
   @ElementCollection(fetch = FetchType.EAGER)
   @CollectionTable(name = "instance_measurements", joinColumns = @JoinColumn(name = "host_id"))
   @MapKeyEnumerated(EnumType.STRING)
-  @MapKeyColumn(name = "uom")
+  @MapKeyColumn(name = "metric_id")
   @Column(name = "value")
   private Map<Measurement.Uom, Double> measurements = new EnumMap<>(Measurement.Uom.class);
```

Comment:
I didn't change the type `Measurement.Uom` by `MetricId` on purpose because the values didn't fully match and I guessed that this was out of scope for this task.
I guess this will be done as part of https://issues.redhat.com/browse/SWATCH-1231.

Reply:
Actually we have another epic for removal of UOM references from the codebase: https://issues.redhat.com/browse/SWATCH-1394 ; the epic will need some updates, but we can scope into https://issues.redhat.com/browse/SWATCH-1379.

Reply:
+1